isis/src/base/objs/TrackingTable/TrackingTable.h

```diff
@@ -0,0 +1,74 @@
+#ifndef TrackingTable_h
+#define TrackingTable_h
+/**
+ * @file
+ * $Revision: 1.3 $
+ * $Date: 2010/05/14 19:17:09 $
+ *
+ *   Unless noted otherwise, the portions of Isis written by the USGS are public
+ *   domain. See individual third-party library and package descriptions for
+ *   intellectual property information,user agreements, and related information.
+ *
+ *   Although Isis has been used by the USGS, no warranty, expressed or implied,
+ *   is made by the USGS as to the accuracy and functioning of such software
+ *   and related material nor shall the fact of distribution constitute any such
+ *   warranty, and no responsibility is assumed by the USGS in connection
+ *   therewith.
+ *
+ *   For additional information, launch
+ *   $ISISROOT/doc//documents/Disclaimers/Disclaimers.html in a browser or see
+ *   the Privacy &amp; Disclaimers page on the Isis website,
+ *   http://isis.astrogeology.usgs.gov, and the USGS privacy and disclaimers on
+ *   http://www.usgs.gov/privacy.html.
+ */
+
+#include "FileName.h"
+#include "Pvl.h"
+#include "Table.h"
+
+#include <QList>
+#include <QString>
+
+
+namespace Isis {
+  /**
+   * @brief Table to store tracking information for a mosaic.
+   *
+   * This table will currently be stored in the label of a separate cube. This tracking cube will
+   * also contain a single tracking band. The DN values stored in this band will correlate to the
+   * indices in this table. Each record within this table will contain the filename of an
+   * associated cube, that cube's serial number, and the DN value associated with this cube within
+   * the tracking band (which should also be its index in the table).
+   *
+   * @author 2018-07-19 Jesse Mapel & Summer Stapleton
+   *
+   * @internal
+   */
+   
+  const QString tableName = "InputImages";
```

Comment:
Because this variable is now a part of the Isis namespace, it should have a more descriptive name. Outside of this class, "tableName" could mean anything. It should be changed to something to like "trackingTableName"